packages/webpack-plugin/src/index.js

```diff
@@ -101,6 +109,26 @@ class LoadablePlugin {
   apply(compiler) {
     this.compiler = compiler
 
+    const { webpack } = compiler
+
+    if (
+      (this.opts.serverSideModuleFederation === true &&
+        compiler.options.target === 'web') ||
+      compiler.options.target === undefined
```

Comment:
these parens make this logic wrong I think? should be
```suggestion
      this.opts.serverSideModuleFederation === true &&
         (compiler.options.target === 'web' || compiler.options.target === undefined)
```

Reply:
Yeah, it was not correct, just fixed, but it was not the problem